lib/src/scrollbar95.dart

```diff
@@ -0,0 +1,383 @@
+import 'dart:async';
+import 'dart:math';
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter95/flutter95.dart';
+
+class Scrollbar95 extends StatefulWidget {
+  final Widget child;
+  final double scrollAmount;
+  final double minThumbHeight;
+  final double scrollbarThickness;
+
+  const Scrollbar95({
+    super.key,
+    required this.child,
+    this.scrollAmount = 0.05,
+    this.scrollbarThickness = 30,
+    this.minThumbHeight = 10,
+  }) : assert(scrollAmount > 0 && scrollAmount <= 1);
+
+  @override
+  State<Scrollbar95> createState() => _Scrollbar95State();
+}
+
+class _Scrollbar95State extends State<Scrollbar95> {
+  final ValueNotifier<double> _relativeTrackPosition =
+      ValueNotifier<double>(0.0);
+  final Duration _scrollingInterval = const Duration(milliseconds: 200);
+  final GlobalKey _trackKey = GlobalKey();
+  ScrollController? _controller;
+  Timer? _continuousScrollingTimer;
+  double _thumbExtent = 30;
+  Axis _scrollDirection = Axis.vertical;
```

Comment:
I think we need a better way to determine the scroll direction. On desktop linux the default of Axis.vertical is never updated. Maybe this is a property that can be obtained from the scroll controller? or the scrollview?